epersongroups.md

```diff
@@ -0,0 +1,307 @@
+# EPerson groups Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**GET /api/eperson/groups**
+
+## Single EPerson Group
+**GET /api/eperson/groups/<:uuid>**
+
+```json
+{
+  "id": "617cf46b-535c-42d5-9d22-327ce2eff6dc",
+  "uuid": "617cf46b-535c-42d5-9d22-327ce2eff6dc",
+  "name": "Administrator",
+  "handle": null,
+  "metadata": {
+  },
+  "permanent": true,
+  "type": "group",
+  "_links": {
+    "subgroups": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc/subgroups"
+    },
+    "epersons": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc/epersons"
+    },
+    "self": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc"
+    }
+  }
+}
+```
+
+## Create new EPerson Group
+
+**POST /api/eperson/groups**
+
+To create a new EPerson Group, perform a post with the JSON below to the epersons endpoint when logged in as admin.
+
+```json
+{
+  "name": "New Group 1",
+  "metadata": {
+      "dc.description": [
+        {
+          "value": "Test group",
+          "language": null,
+          "authority": "",
+          "confidence": -1
+        }
+      ]
+  }
+}
+```
+
+Status codes:
+* 201 Created - if the operation succeed
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions
+* 422 Unprocessable Entity - if the name was omitted or already exists, if permanent was set to true
```

Comment:
@tdonohue how I see this is:
`permanent` is a property on the group which is part of the JSON response. It's possible this is set to true during the POST (Angular should not be implemented to set it to true, but it can still be sent as such to REST from other applications). Since we should not allow creating permanent groups via REST, it should fail

Reply:
@benbosman : Oh, I understand what you mean now. You are correct, but I previously misunderstood the statement without the additional context you just added.  I'd suggest we simply reword this slightly and say: 

`422 Unprocessable Entity - if the name was omitted or already exists, *or* if permanent was set to true (permanent groups can only be created by the system).`